src/components/FilterPanel.js

```diff
@@ -233,11 +233,12 @@ export default function FilterPanel(props) {
                             // Is the location within the range specified?
                             return d.miles < zipCodeFilter.miles;
                         }
-                    } else {
-                        setSortBy("location");
                     }
                 }
-                return true; // include all for now.
+                // No zipcode was provided or
+                // was unable to find coordinates for the specified zipcode
+                setSortBy("location");
```

Comment:
So, if I type in "treehouse," the filter doesn't show me all the vax sites near my treehouse.
This is unreasonable, but what can you do?

Nevertheless, there's no indication or feedback that it even _tried_. I'm not sure what to do about that, but it seems like there should be … something.

Reply:
I agree, but was unsure _where_ to display such an error message.  This is also a problem in the interim case of while you are entering a ZIP Code and stop at "020"

Reply:
@liv Maybe you can incorporate this idea while you are working on the FilterPanel.   Since the panel is non-modal, we need someway to signify that the ZIP Code is **not valid**.  This might be because we don't have a **long/lat** for it, or maybe its not well-formed **/\d{5}/**